packages/infra/lib/api-stack.ts

```diff
@@ -558,6 +559,7 @@ export class APIStack extends Stack {
     patientCreateLambda.addEnvironment("API_URL", `http://${apiDirectUrl}`);
     patientQueryLambda.addEnvironment("API_URL", `http://${apiDirectUrl}`);
 
+    // TODO move this to each place where it's used
```

Comment:
what does this mean?

Reply:
we're exposing the connection between the bucket and the API on the stack code. That sort of code should be on the API setup logic, it knows whether it needs read, write, or read/write access to the bucket. This makes the stack code much more complex than it needs to be.